pkg/console/subresource/deployment/deployment.go

```diff
@@ -142,6 +144,18 @@ func DefaultDeployment(operatorConfig *operatorv1.Console, cm *corev1.ConfigMap,
 							Operator: corev1.TolerationOpExists,
 							Effect:   corev1.TaintEffectNoSchedule,
 						},
+						{
```

Comment:
Please check, if this is needed or not@spadgett

Reply:
This seems reasonable to me.